scripts/variantstore/wdl/extract/vds_validation.py

```diff
@@ -42,18 +42,25 @@ def check_densify_small_region(vds):
 
 
 def main(vds):
-    check_samples_match(vds)
-    check_ref_blocks(vds)
     check_densify_small_region(vds)
-    vds.validate(); print('Hail VDS validation successful')
+    chrs = [f'chr{c}' for c in itertools.chain(range(1, 23), ['X', 'Y', 'M'])]
+    for chromosome_to_validate in chrs:
+        filtered_vd = vds.variant_data.filter_rows(vds.variant_data.locus.contig == chromosome_to_validate)
+        filtered_rd = vds.reference_data.filter_rows(vds.reference_data.locus.contig == chromosome_to_validate)
+        filtered_vds = hl.vds.VariantDataset(filtered_rd, filtered_vd)
+        print(f"Validating VDS chromosome {chromosome_to_validate}...")
+        check_ref_blocks(filtered_vds)
+        print(f"Hail VDS validation successful for chromosome {chromosome_to_validate}")
```

Comment:
these log messages should maybe be a bit more specific as to what's being validated; as they are now it sort of sounds like everything is good for a given chromosome, but there are other validations remaining outside the loop that could still fail